projectPage.css

```diff
@@ -1,57 +1,36 @@
-.vid {
+@import url('https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Oswald:wght@500&display=swap');
+
+
+/* HEADER 
+=================================================*/
+.header-vid {
     object-fit: cover;
-    width: 100vw;
+    width: 100%;
     height: 200px;
     position: fixed;
-   
   }
 
-  /* @media screen and (max-width: 600px) {
-    #vid {
-      width: 36%;
-      height: 7%;
-    }
-  } */
-
 
-h1 {
+.header-h1 {
+    /* Text positioning */
     position: fixed;
-    flex-direction: row;
+    margin-top: auto;
+    width: 100%;
+    padding: 0;
+
+    /* Text formatting */
     text-shadow: 6px 6px 6px   rgb(53, 50, 50) ;
     color: #fff;
     font-family: 'Oswald', sans-serif;
-    /* mix-blend-mode:overlay; */
-    /* mix-blend-mode: multiply; */
-    margin-top: auto;
-    margin-bottom: auto;
-    font-size: 115px;
-    margin-left: auto;
-    margin-right: auto;
-    height: 20%;
-    width: 100%;
+    font-size: 7rem;
+    line-height: 200px;
```

Comment:
setting line height of text equal to the height of the container will center the text on the vertical axis. in other words if you want your text to be centered in a box, you can center it horizontally using text-align: center and you can center it vertically by using line-height: [height of container]